org.openhab.binding.zigbee.ember/src/main/java/org/openhab/binding/zigbee/ember/internal/discovery/ZigBeeEmberUsbSerialDiscoveryParticipant.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2014-2018 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.zigbee.ember.internal.discovery;
+
+import static java.util.Arrays.asList;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_BAUD;
+import static org.openhab.binding.zigbee.ZigBeeBindingConstants.CONFIGURATION_PORT;
+import static org.openhab.binding.zigbee.ember.EmberBindingConstants.THING_TYPE_EMBER;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.smarthome.config.discovery.DiscoveryResult;
+import org.eclipse.smarthome.config.discovery.DiscoveryResultBuilder;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDeviceInformation;
+import org.eclipse.smarthome.config.discovery.usbserial.UsbSerialDiscoveryParticipant;
+import org.eclipse.smarthome.core.thing.ThingTypeUID;
+import org.eclipse.smarthome.core.thing.ThingUID;
+import org.osgi.service.component.annotations.Component;
+
+/**
+ * Discovery for ZigBee USB dongles, integrated in Eclipse SmartHome's USB-serial discovery by implementing
+ * a component of type {@link UsbSerialDiscoveryParticipant}.
+ * <p/>
+ * Currently, this {@link UsbSerialDiscoveryParticipant} supports the ZigBee dongle 'BV AV2010/10' from Bitron Video.
+ *
+ * @author Henning Sudbrock - initial contribution
+ */
+@Component(service = UsbSerialDiscoveryParticipant.class)
+public class ZigBeeEmberUsbSerialDiscoveryParticipant implements UsbSerialDiscoveryParticipant {
+
+    private static final Set<ThingTypeUID> SUPPORTED_THING_TYPES = new HashSet<>(asList(THING_TYPE_EMBER));
+
+    public static final int SILICON_LABS_USB_VENDOR_ID = 0x10c4;
+    public static final int BITRON_VIDEO_2010_10_PRODUCT_ID = 0x8b34;
+    private static final int BITRON_VIDEO_2010_10_BAUD_RATE = 57600;
+    private static final String BITRON_VIDEO_2010_10_DEFAULT_LABEL = "Bitron Video AV2010/10 Ember ZigBee Dongle";
+
+    @Override
+    public Set<ThingTypeUID> getSupportedThingTypeUIDs() {
+        return SUPPORTED_THING_TYPES;
+    }
+
+    @Override
+    public @Nullable DiscoveryResult createResult(UsbSerialDeviceInformation deviceInformation) {
+        if (isBitronVideoDongle(deviceInformation)) {
+            return DiscoveryResultBuilder.create(createBitronVideoDongleThingType(deviceInformation))
+                    .withLabel(createBitronVideoDongleLabel(deviceInformation))
+                    .withRepresentationProperty(CONFIGURATION_PORT)
+                    .withProperty(CONFIGURATION_PORT, deviceInformation.getSerialPort())
+                    .withProperty(CONFIGURATION_BAUD, BITRON_VIDEO_2010_10_BAUD_RATE).build();
```

Comment:
It may also be worth setting the flow control property as well (Should be XON/OFF for this version).

Reply:
Ah, I wasn't aware of that option yet, thanks! I will add this with my next commit to this PR (I tested this setting with the Ember dongle, and it worked fine so far).

@cdjackson Do you have any infos on flow control for the telegesis dongles? I have looked at https://www.silabs.com/documents/public/reference-manuals/TG-PM-0518-ETRX357USB.pdf which mentions that nRTS and nCTS lines are connected to the USB bridge chip - from which I (as a layman) would deduce that hardware flow control could be used - or am I mistaken completely here?

Reply:
> I tested this setting with the Ember dongle, and it worked fine so far

I think the lower speed that is used in the XON/OFF flow control means that it's not a big issue this way around when running on a reasonably fast computer. But the other way around (ie using XON/OFF instead of CTS/RTS) can completely stop the dongle from working. This is what is used when the Ember runs at 115k2.

> which mentions that nRTS and nCTS lines are connected to the USB bridge chip - from which I (as a layman) would deduce that hardware flow control could be used - or am I mistaken completely here?

I would agree with your assessment. Currently I think I don't use flow control. I think I read somewhere that it is optional, but the Telegesis runs at such a slow speed (19k2) that it shouldn't matter I think (except on a really slow computer!)

Reply:
Thanks for explaining! I wil thenl leave it as is for now (i.e., use the default flow control setting for the telegesis discovery result, which is 'no flow control'). We can still change this if issues arise somewhere.